hrjobcontract/CRM/Hrjobcontract/Import/Parser/BaseClass.php

```diff
@@ -82,8 +82,10 @@ function init() {
       $field['headerPattern'] = CRM_Utils_Array::value('headerPattern', $field, '//');
       $this->addField($name, CRM_Utils_Array::value('title', $field, $name), $field['type'], $field['headerPattern'], $field['dataPattern']);
     }
+
+    $mapperLocType = !empty($mapperLocType) ? $mapperLocType : [];
     $this->setActiveFields($this->_mapperKeys);
-    $this->setActiveFieldLocationTypes($this->_mapperLocType);
```

Comment:
Since this line was removed, is `$this->_mapperLocType` still being used in other classes? If not i guess we can remove the declaration 🤷‍♂️

Reply:
Oops. It wasn't meant to be removed. Added back. Nice catch!